### PR TITLE
Added a C++ code that prints the decimal equivalent of a binary string

### PR DESCRIPTION
**Pull request by soukarsha122**
Issue #170

### CODE REVIEW

**Comment by soukarsha122** [comment]
@tarun26091999 Sir please review the code and tell me if I am wrong anywhere. It will also be very helpful if you please say which readme I have to update , is it the string readme or the readme of number theory ....

**Comment by soukarsha122** [comment]
@plazzy99 @rudrakshi99 @plazzy99 please review my code

**Comment by soukarsha122** [comment]
@raksha009  @shreyanspoddar please review my pr

**Comment by soukarsha122** [comment]
@OjusWiZard @tarun26091999 @plazzy99 pls review my pr

**Comment by soukarsha122** [comment]
@plazzy99 @rudrakshi99 please tell me which readme I have to update? Is it the string readme?

**Comment by yasharth291** [comment]
please change the file readme.md mentioning your code @soukarsha122

**Comment by soukarsha122** [comment]
@yasharth291 I have done the changes as requested...Please review my changes...Please tell me if any more changes needs to be done..
Please merge this pr...Thank you very much

**Comment by soukarsha122** [comment]
@plazzy99 @rudrakshi99 @yasharth291 please review my pr...please merge it

File: string/readme.md [hunk 1]
```diff
@@ -13,10 +13,19 @@ A string is a class that contains a char array, but automatically manages it for
...
+
+* Print the decimal equivalent of a binary string ---->[C++](/Code/C++/binary_string_to_decimal.cpp)
+
 * Convert string to lowercase and uppercase ---->[C++](/Code/C++/string_to_lowercase_and_uppercase.cpp)
 * Manacher's algorithm (finding the longest palindrome in a string) ----> [Java](/Code/Java/longPalindrome.java)
 * Maximum occuring character in a string ---->[C++](/Code/C++/max_count.cpp)
 * Number of deletions to make a string pallindrome ---->[Python](/string/Number_of_deletions_to_make_pallindrome.py)
+
 * Revere a string using Stack ---->[C++](/Code/C++/reverse_a_string_using_stack.cpp)
 * Reverse Individual Words in String ----> [Java](/Code/Java/revindivstring.java)
```
Comment (rish-singhal) [inline string/readme.md:25]: sort this list in lexicographic order

File: Code/C++/binary_string_to_decimal.cpp [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/*
+
+A string containing only 0's and 1's is called a binary string.
+
+The equivalent decimal of a binary number is the sum of 2 raised to the power of the positional values where 1 is present. 
+The least significant bit is assigned positional value 0.
+
+The following program illustrates how to convert a binary string to its equivalent decimal
+
```
Comment (rish-singhal) [inline Code/C++/binary_string_to_decimal.cpp:9]: remove extra line

File: Code/C++/binary_string_to_decimal.cpp [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Sample input : 1100
+Standard output: 12
+
+Sample input : 1000
+Standard output: 8
+
+Sample input : 100010
+Standard output: 34
+
+Time Complexity : O(n)
```
Comment (rish-singhal) [inline Code/C++/binary_string_to_decimal.cpp:57]: describe what is n here

File: Code/C++/binary_string_to_decimal.cpp [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+	// Running a loop to convert the binary to its equivalent decimal
+	for(int i=n-1;i>=0;i--)
+	{
+		// Updating the variable decimal if the ith position of the string has value '1'
+		if(s[i]=='1')
+		decimal+=(int)pow(2,abs(i-(n-1)));
+	}
+	
+	// Printing the equivalent decimal of the inputted binary string
+	cout<<decimal;
```
Comment (rish-singhal) [inline Code/C++/binary_string_to_decimal.cpp:38]: give appropriate space aorund "<<, ==, +=, >= ...."

File: Code/C++/binary_string_to_decimal.cpp [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+Sample input : 100010
+Standard output: 34
+
+Time Complexity : O(n)
+Space Complexity: O(n)
+
+*/
+
+
+
```
Comment (rish-singhal) [inline Code/C++/binary_string_to_decimal.cpp:63]: remove these extra lines

File: string/readme.md (hunk 1 above)
Comment (soukarsha122) [reply string/readme.md:25]: @rish-singhal  Sir I have done as asked...just one question..there are two duplicate codes in this readme ..will I remove the duplicates?

**Comment by soukarsha122** [comment]
@rish-singhal I have done changes as requested. But I found there are two duplicates in readme . Will I remove the duplicates?

**Comment by soukarsha122** [comment]
@rish-singhal I have done the required changes. Please tell me if anything more I have to change

**Pull request merged**